packages/fungible-token/bridge-fungible-token/tests/utils/interface/bridge/mod.rs

```diff
@@ -42,29 +45,29 @@ pub(crate) async fn withdraw(
         .unwrap()
 }
 
-pub(crate) async fn bridged_token(
-    contract: &BridgeFungibleTokenContract<WalletUnlocked>,
-) -> Bits256 {
-    contract
-        .methods()
-        .bridged_token()
-        .call()
-        .await
-        .unwrap()
-        .value
-}
+// pub(crate) async fn bridged_token(
```

Comment:
Are these still needed?

Reply:
No, this methods are gone. Will remove the commented out code